scripts/update/Updater.php

```diff
@@ -253,5 +254,9 @@ public function update($initialVersion)
         }
 
         $this->skip('8.2.0', '8.3.0');
+        if ($this->isVersion('8.3.0')) {
+            $this->runExtensionScript(RegisterProctoringLtiMessageFactory::class);
```

Comment:
we discussed it before: we can't use install script in the updater, because it makes possible to have different state of the environment on the same version on the different servers

Reply:
Fixed